nuxeo-jsf/nuxeo-platform-webapp-base/src/main/resources/OSGI-INF/restAPI-contrib.xml

```diff
@@ -2,13 +2,13 @@
 <component name="org.nuxeo.ecm.webapp.seam.debug.restAPI.contrib">
 
   <extension target="org.nuxeo.ecm.platform.ui.web.restAPI.service.PluggableRestletService"
-      point="restlets">
+             point="restlets">
```

Comment:
Only cleanup, not needed.

Reply:
I took the hypothesis that I had to format the wholes files (java classes or xml) where a class will be impacted as example the class org.nuxeo.ecm.platform.ui.web.auth.service.PluggableAuthenticationService  is impacted in java code. And then I did the xml file 

i will pay attention about the separation and formatting only the affected files only